buildout.cfg

```diff
@@ -1,60 +1,8 @@
 [buildout]
+develop = .
 parts =
-    zope2
-    instance
-    zopepy
     test
-    coverage-test
-    coverage-report
-
-eggs =
-extends = http://dist.plone.org/release/3.2.1/versions.cfg
-develop = .
-
-[zope2]
-recipe = plone.recipe.zope2install
-url = http://www.zope.org/Products/Zope/2.11.2/Zope-2.11.2-final.tgz
-
-[instance]
```

Comment:
Is it maybe worth to add this instance part to the readme? Especially the inclusion of the `zcml=` might be a source of errors.

Reply:
It was already in docs/INSTALL.rst which I included into README to improve its visibility.